src/classes/updates-reset-stale.ts

```diff
@@ -0,0 +1,90 @@
+import {Option} from '../enums/option';
+import {IIssuesProcessorOptions} from '../interfaces/issues-processor-options';
+import {Issue} from './issue';
+import {IssueLogger} from './loggers/issue-logger';
+
+export class UpdatesResetStale {
+  private readonly _options: IIssuesProcessorOptions;
+  private readonly _issue: Issue;
+  private readonly _issueLogger: IssueLogger;
+
+  constructor(options: Readonly<IIssuesProcessorOptions>, issue: Issue) {
+    this._options = options;
+    this._issue = issue;
+    this._issueLogger = new IssueLogger(issue);
+  }
+
+  shouldUpdatesResetStale(): boolean {
+    return this._shouldUpdatesResetStale();
+  }
+
+  private _shouldUpdatesResetStale(): boolean {
+    return this._issue.isPullRequest
+      ? this._shouldIgnoreAllPullRequestActivitiesBeforeStale()
+      : this._shouldIssueUpdatesResetStale();
+  }
+
+  private _shouldIgnoreAllPullRequestActivitiesBeforeStale(): boolean {
```

Comment:
this this method be renamed too?

Reply:
Yeah it's still a WIP

Reply:
b903c26aebae1666d738aba923dff7ff6129602f